charts/cp-schema-registry/values.yaml

```diff
@@ -86,5 +120,12 @@ prometheus:
     imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
     imagePullPolicy: IfNotPresent
     port: 5556
+  serviceMonitor: # defines monitoring for a service when using prometheus operator
+    enabled: true
+    jobLabel: app
 
     resources: {}
+
+ui:
```

Comment:
It seems that there are no resources (requests/limits) for the ui.
You should add them here in `values.yaml` and in `templates/schema-registry-ui.yaml`

Reply:
Good to have added the resources.
Are you sure that the 10Mi default memory request is enough ? It looks small, even for "just" a UI.

Reply:
I picked these values from the actual resources in kubernetes, it has been running like this for months so I didn't change them.

Reply:
and the running pods are using how much ram ?

Reply:
Seems to be consistent with the request 
- [platform](https://grafana.prod-1.blbl.cr/d/wzrL-WWMk/03-application-cost-optimisation?viewPanel=1&orgId=1&var-project=infra-prod-67bbc6f2&var-namespace=platform&var-app=platform-schema-registry-cp-schema-registry&var-container=platform-schema-registry-cp-schema-registry-ui&var-resolution=$__auto_interval_resolution&from=now-30d&to=now)
- [feature-store](https://grafana.prod-1.blbl.cr/d/wzrL-WWMk/03-application-cost-optimisation?viewPanel=1&orgId=1&from=now-30d&to=now&var-project=infra-prod-67bbc6f2&var-namespace=feature-store&var-app=franz-kafka-schema-registry-cp-schema-registry&var-container=franz-kafka-schema-registry-cp-schema-registry-ui&var-resolution=$__auto_interval_resolution)

Reply:
I see that the memory working-set can go up to ~15 MB on your graph, so more than the current 10MB request.
So, we should take no risk and increase the request to 30MB or 50MB (those are examples, the values are so small that cost will be minimal, so let's put some margin)

Reply:
👍